src/org/jitsi/impl/neomedia/rtcp/RTCPTCCPacket.java

```diff
@@ -605,44 +627,42 @@ else if (tsDelta < 8191 && tsDelta > -8192)
             // symbol (we've already set 'off' to point to the correct byte).
             //  0 1 2 3 4 5 6 7          8 9 0 1 2 3 4 5
             //  S T <0> <1> <2>          <3> <4> <5> <6>
-            int symbolOffset;
+            int symbolShift;
```

Comment:
I remember we discovered a bug in here, but now that I look at it again I can not find it. Was it just a fictitious bug?

Reply:
Yeah, and our fictitious fix behaved worse than the original version.